falcon/testing/client.py

```diff
@@ -808,6 +814,11 @@ async def _simulate_request_asgi(
             overrides `body` and sets the Content-Type header to
             ``'application/json'``, overriding any value specified by either
             the `content_type` or `headers` arguments.
+                msgpack(Msgpack serializable): A Msgpack document to serialize as the
```

Comment:
I think this section is over indented, also the other sections in the other docstrings